examples/MongoDB-Atlas-AWS-VPC-Peering/atlas.tf

```diff
@@ -0,0 +1,71 @@
+provider "mongodbatlas" {
+  public_key  = var.public_key
+  private_key = var.private_key
+}
+
+resource "mongodbatlas_project" "aws_atlas" {
+  name   = "aws-atlas"
+  org_id = var.atlasorgid
+}
+
+resource "mongodbatlas_cluster" "cluster-atlas" {
+  project_id   = mongodbatlas_project.aws_atlas.id
+  name         = "cluster-atlas"
+  cluster_type = "REPLICASET"
+  replication_specs {
+    num_shards = 1
+    regions_config {
+      region_name     = "US_EAST_1"
+      electable_nodes = 3
+      priority        = 7
+      read_only_nodes = 0
+    }
+  }
+  provider_backup_enabled      = true
+  auto_scaling_disk_gb_enabled = true
+  mongo_db_major_version       = "4.2"
```

Comment:
Just a comment for later but we should try to make this always the latest version. Not a blocker though.

Reply:
fixed